frontend/sw360-portlet/src/main/resources/META-INF/resources/html/utils/includes/searchReleases.jsp

```diff
@@ -177,5 +181,27 @@
             });
         }
 
+        function makeReleaseViewUrl(releaseId) {
+            return homeUrl + '/components/-/component/release/detailRelease/' + releaseId;
+        }
+
+        function detailUrl(name, releaseId)
+        {
+            let url = makeReleaseViewUrl(releaseId);
+            let viewUrl = $("<a></a>").attr("href",url).attr("target","_blank").css("word-break","break-word").html(name);
+            return viewUrl[0].outerHTML;
+        }
+
+        function addLinkToReleaseNameAndVersion() {
+            $('#releaseSearchResultsTable > tbody  > tr').each(function() {
+                var $releasId = $('td:eq(0)', this).find('input').val();
+                var $relName = $('td:eq(2)', this);
+                var $relVersion = $('td:eq(3)', this);
+                let linkOnRelName = detailUrl($relName.text(), $releasId);
```

Comment:
Both `Name` and `Version` hyperlink is opening `Release` details page.
Ideally. `Name` should open the component details page. 
Kindly fix it.

Reply:
Fixed it along with renaming the column name to `Component Name` and `Release Version` respectively.